bin/k8s-read-config

```diff
@@ -34,6 +34,9 @@ fi
 echo "[dockerfile]: $DOCKERFILE"
 echo "[dockertag]: $DOCKERTAG"
 
+NAMESPACE=${NAMESPACE:=default}
```

Comment:
I'm guessing you mean breaking if one does not already specify `NAMESPACE` in their config? I thought setting the default here to its current assumed value would take care of that case?